metron-platform/metron-solr/src/main/java/org/apache/metron/solr/dao/SolrDao.java

```diff
@@ -72,12 +72,12 @@ public SolrDao() {
   @Override
   public void init(AccessConfig config) {
     if (config.getKerberosEnabled()) {
-      HttpClientUtil.addConfigurer(new Krb5HttpClientConfigurer());
+      enableKerberos();
```

Comment:
Thank you for the refactorings like this.  Even though small, they help with readability a bunch.  Bravo.